.travis.yml

```diff
@@ -1,23 +1,31 @@
 language: python
-
 python:
-  - "3.5"
-
+- 3.5
+- 3.6
 env:
-  - DJANGO=""   # no django
-  - DJANGO="Django>=1.11,<1.12"
-
+- DJANGO=""
+- DJANGO=1.11
+- DJANGO=2.0
+- DJANGO=2.1
+- DJANGO=2.2
+- DJANGO=3.0
+jobs:
+  exclude:
+  - python: 3.5
+    env: DJANGO=3.0
```

Comment:
is this because django 3.0 is py3.6 and above only?

Reply:
Correct — see https://docs.djangoproject.com/en/3.0/releases/3.0/#python-compatibility

(They actually "highly recommend and only officially support" the latest version of python, i.e. 3.8, but we don't have a python version upgrade path for our production environments yet.)